tests/argv/enc/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+oeedl_file(../argv.edl enclave gen)
+
+add_enclave(TARGET argv_enc
+    UUID 71b0822f-42a3-4543-a97c-ca491f76b82c SOURCES enc.c ${gen})
```

Comment:
Good find. The use of UUID troubles me. I'm working with Hernan and will try to find a better solution so we don't have to maintain all these UUID's. This will continue to be a problem.